src/main/kotlin/org/web3j/evm/EmbeddedWeb3jService.kt

```diff
@@ -45,6 +45,9 @@ import org.web3j.utils.Numeric
 class EmbeddedWeb3jService(configuration: Configuration, operationTracer: OperationTracer) : Web3jService {
     private val embeddedEthereum: EmbeddedEthereum = EmbeddedEthereum(configuration, operationTracer)
 
+    constructor(configuration: Configuration) : this(configuration, PassthroughTracer()) {
```

Comment:
Looking good, thanks!

In Kotlin you don't actually need the empty { and } btw